app/views/google/results-wi.html

```diff
@@ -0,0 +1,8 @@
+<html>
+    <head>
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+      <a href="/ae-finder"><img src="/assets/images/google-search/google-results-ae.png" style="width: 100%; display: block;"></a>
```

Comment:
Do we need a walk-in centre image for this rather?

Reply:
And should this go to the walk-in-finder?

Reply:
Yeah, still working on the google pages atm, these changes were to just set up different links on the index page. One for walk in one for A&E